third-culture-software/bhima
# Updated algorithm names

This PR updated the CMM/AMC algorithm names from "Algorithm 1" and "Algorithm 4 (MSH)" to "Default Algorithm" and "MSH Algorithm" respectively.   It also cleans out some left-over code related to algorithms 2 and 3, including some end-to-end tests that might have been broken before this change.  All of the tests work on my system--except the end-to-end test which fails on my system exactly like the master branch has been doing for some time.

Closes https://github.com/IMA-WorldHealth/bhima/issues/5498

**Testing**
- Try the Articles in Stock page and view the CMM calculation dialog (double-click on CMM on a row)
- Try changing the CMM algorithm in the Stock Settings page and verify that it all looks good.
- Go back to the Articles in Stock page and verify that the algorithm changed worked.
- Please verify that the French translations are okay.
